lib/metrics.js

```diff
@@ -136,26 +140,39 @@ Histogram.prototype.addPoint = function(val, timestampInMillis) {
     this.samples.push(val);
 };
 
-Histogram.prototype.flush = function() {
-    const points = [
-        this.serializeMetric(this.min, 'gauge', this.key + '.min'),
-        this.serializeMetric(this.max, 'gauge', this.key + '.max'),
-        this.serializeMetric(this.sum, 'gauge', this.key + '.sum'),
-        this.serializeMetric(this.count, 'count', this.key + '.count'),
-        this.serializeMetric(this.average(), 'gauge', this.key + '.avg')
-    ];
-
+Histogram.prototype.flush = function () {
+    let points = [];
+    if (this.aggregates.indexOf("min") !== -1) {
+      points.push(this.serializeMetric(this.min, "gauge", this.key + ".min"));
```

Comment:
Nit: prefer single quotes for consistency with existing code

Reply:
Make sense 👍 . I changed all the `"` by `'`